data/outreachy.yml

```diff
@@ -172,9 +172,11 @@ rounds:
           for example, in the form of macros or templates. The OCaml compiler
           doesn't provide a full built-in macro system, but the OCaml parser does
           provide syntax for preprocessing purposes: attributes
-          (https://v3.ocaml.org/docs/metaprogramming#attributes-and-derivers) and
+          (https://v3.ocaml.org/docs/metaprogramming
+          #attributes-and-derivers) and
           extension points
-          (https://v3.ocaml.org/docs/metaprogramming#extension-nodes-and-extenders).
+          (https://v3.ocaml.org/docs/metaprogramming
+          #extension-nodes-and-extenders).
```

Comment:
I think, since these descriptions are just plain text, they cannot contain URLs in the first place. So the best thing here would be to remove the links.

Reply:
That is fine then.